.changeset/bright-papayas-speak.md

```diff
@@ -0,0 +1,7 @@
+---
+"@finos/legend-art": minor
```

Comment:
Only `@finos/legend-graph`, `@finos/legend-studio` and `@finos/legend-extension-external-language-morphir` should be patch.

Reply:
Changed modules to patch change.